tests/org.codechecker.eclipse.rcp.it.tests/pom.xml

```diff
@@ -13,37 +13,177 @@
     <version>1.0.0-SNAPSHOT</version>
     <packaging>eclipse-test-plugin</packaging>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.eclipse.tycho</groupId>
-                <artifactId>target-platform-configuration</artifactId>
-                <version>${tycho.version}</version>
-                <configuration>
-                    <dependency-resolution>
-                        <extraRequirements>
-                            <requirement>
-                                <type>eclipse-feature</type>
-                                <id>org.eclipse.cdt</id>
-                                <versionRange>0.0.0</versionRange>
-                            </requirement>
-                        </extraRequirements>
-                    </dependency-resolution>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.eclipse.tycho</groupId>
-                <artifactId>tycho-surefire-plugin</artifactId>
-                <version>${tycho.version}</version>
-                <configuration>
-                    <useUIHarness>true</useUIHarness>
-                    <useUIThread>false</useUIThread>
-                    <product>org.codechecker.eclipse.feature</product>
-                    <application>org.eclipse.ui.ide.workbench</application>
-                    <testClass>org.codechecker.eclipse.plugin.AllTests</testClass>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+    <profiles>
```

Comment:
We have to use different configurations depending on the jdk verisons. Maven profiles can only alter build segments, thats why this segment got a little bit verbose.